cemu/ui/codeeditor.py

```diff
@@ -34,30 +33,22 @@
 
 
 class CodeEdit(QTextEdit):
-    def __init__(self, parent):
+    def __init__(self, parent=None):
```

Comment:
Can you add type hints too?
```suggestion
    def __init__(self, parent: Optional[QtWidget] = None):
```